backend/main.py

```
@@ -1,369 +1,407 @@
-    def __init__(self):
-        # rooms: { "room_id": [WebSocket, WebSocket, ...] }
-        self.active_connections: Dict[str, List[WebSocket]] = {}
+from fastapi import WebSocket, WebSocketDisconnect
 
+class ChatConnectionManager:
+    def __init__(self): self.active_connections: Dict[str, List[WebSocket]] = {}
     async def connect(self, websocket: WebSocket, room_id: str):
         await websocket.accept()
-        if room_id not in self.active_connections:
-            self.active_connections[room_id] = []
+        if room_id not in self.active_connections: self.active_connections[room_id] = []
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)